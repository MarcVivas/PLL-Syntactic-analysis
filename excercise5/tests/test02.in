/Error on line 1
S : aAc | fA | Bdef
;
B : Ab | Ad | k
;
A : ilm | ml
;
