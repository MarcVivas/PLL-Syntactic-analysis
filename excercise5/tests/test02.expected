Error at line 1: syntax error
Acabament fitxer inesperat LÃ­nea 1
